src/test/java/com/exasol/errorreporting/ErrorMessageBuilderTest.java

```diff
@@ -0,0 +1,54 @@
+package com.exasol.errorreporting;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class ErrorMessageBuilderTest {
+
+    @Test
+    void testMessage() {
+        final String message = new ErrorMessageBuilder("T-ERJ-1").message("Test message.").toString();
+        assertThat(message, equalTo("T-ERJ-1: Test message."));
+    }
+
+    @Test
+    void testMessageWithPlaceholder() {
+        final String message = new ErrorMessageBuilder("T-ERJ-1")
+                .message("Test message {{myPlaceholder}} and a number {{number}}.")
+                .parameter("myPlaceholder", "myValue").parameter("number", 1, "a number").toString();
+        assertThat(message, equalTo("T-ERJ-1: Test message 'myValue' and a number 1."));
+    }
+
+    @Test
+    void testMessageWithUnquotedPlaceholder() {
```

Comment:
maybe we should rename it to `testMessageWithUnquotedParameter` ? Same the other ones